ros/lib/constants.py

```diff
@@ -32,3 +27,57 @@ class CloudProvider(Enum):
 class OperatingSystem(Enum):
     """Supported Operating Systems"""
     RHEL = 'RHEL'
+
+
+class SystemStatesWithKeys(Enum):
+    SYSTEM_STATES = {
+        "INSTANCE_OVERSIZED": "Oversized",
+        "INSTANCE_UNDERSIZED": "Undersized",
+        "INSTANCE_IDLE": "Idling",
+        "INSTANCE_OPTIMIZED_UNDER_PRESSURE": "Under pressure",
+        "STORAGE_RIGHTSIZING": "Storage rightsizing",
+        "OPTIMIZED": "Optimized",
+        "NO_PCP_DATA": "Waiting for data"
+    }
+    OPTIMIZED_SYSTEM = "OPTIMIZED"
+
+
+class RosSummary(Enum):
+    ROSSUMMARY = dict(
+        OPTIMIZED='System is OPTIMIZED',
+        MEMORY_OVERSIZED='Memory utilization is very low',
+        MEMORY_UNDERSIZED='Memory utilization is too high',
+        MEMORY_UNDERSIZED_BY_PRESSURE='System is suffering from memory pressure',
+        CPU_OVERSIZED='CPU utilization is very low',
+        CPU_UNDERSIZED='CPU utilization is too high',
+        CPU_UNDERSIZED_BY_PRESSURE='System is suffering from CPU pressure',
+        IO_OVERSIZED='I/O utilization is very low',
+        IO_UNDERSIZED='I/O utilization is too high',
+        IO_UNDERSIZED_BY_PRESSURE='System is suffering from IO pressure',
+        IDLE='System is IDLE',
+    )
+
+
+class SystemsTableColumn(Enum):
+    """Common column names for System table"""
+    SYSTEM_COLUMNS = [
+        'inventory_id',
+        'display_name',
+        'instance_type',
+        'cloud_provider',
+        'state',
+        'fqdn',
+        'operating_system',
+        'groups'
+    ]
+
+
+# recommendations
+NEWLINE_SEPARATOR = '\n'
+RULES_COLUMNS = ['rule_id', 'description', 'reason', 'resolution', 'condition']
+INSTANCE_PRICE_UNIT = 'USD/hour'
+
+# notification
+BUNDLE = "rhel"
+APPLICATION = "resource-optimization"
+EVENT_TYPE = "new-suggestion"
```

Comment:
I understand these are few vars, however we can still go ahead with separate Enum class for these to maintain the consistency?